pyproject.toml

```diff
@@ -38,8 +38,11 @@ Documentation = "https://github.com/hamdanal/rich-argparse#rich-argparse"
 Issue-Tracker = "https://github.com/hamdanal/rich-argparse/issues"
 Changelog = "https://github.com/hamdanal/rich-argparse/blob/main/CHANGELOG.md"
 
-[tool.hatch]
-build.packages = ["rich_argparse"]
+[tool.hatch.build.targets.sdist]
+exclude = ["/.github", "/.vscode", "/scripts", "/.pre-commit-config.yaml", "dist*"]
```

Comment:
Wouldn't this include everything in the project except for these? I have a lot of cruft in my local project that I wouldn't want to be included in the sdist. Can't we have an explicit _include list_ instead?

Reply:
Sure. I will try an `include` list, it didn't work for me earlier, but maybe I had made some other mistake.

Reply:
Done, it worked this time!